sips/sip-02x/sip-02x-stacks-pay.md

```diff
@@ -0,0 +1,343 @@
+
+**Stacks Pay** addresses this gap by introducing a payment request standard tailored specifically for the Stacks blockchain. The standard simplifies the process of sending and receiving payments, enhances interoperability among wallets and applications, and increases security by providing mechanisms to tie transactions back to payment requests.
+
+Stacks Pay does _not_ define smart contracts that may be used to facilitate specific functions, like subscriptions or NFT mints. Stacks Pay is strictly focused on defining the transaction payment information structure and encoding.
+
+## Operations
+
+Stacks Pay defines several standard operations that specify the type of payment action to be performed. Each operation type has specific required and optional parameters.
+
+### `support`
+
+**Function**: An open-ended, reusable request. Allows the payer to specify the amount and optionally the token, suitable for donations, tips, or gifts.
+
+**Required Parameters:**
+
+- operation: 'support'
+
+- recipient: A valid Stacks address.
+
+- If the token parameter is not 'STX', the functionName defaults to 'transfer' by the wallet.
+
+**Optional Parameters:**
+
+- token: Either 'STX' or a valid SIP-010 contract address. If not provided, defaults to 'STX'.
+
+- description: MAY be included to provide additional context for the payment request.
+
+- expiresAt: MAY be included; wallets MUST NOT process payment requests past the expiry time.
+
+- memo: MAY be included; MUST NOT include personal information.
+
+**Parameters to Ignore:**
+
+- amount, contractName, dueDate, functionName, invoiceNumber. Wallets MUST ignore these parameters if present.
+
+### `invoice`
+
+**Function:** Represents a payment request, such as an invoice for P2P payments or product purchases.
+
+**Required Parameters:**
+
+- operation: 'invoice'
+
+- recipient: A valid Stacks address.
+
+- token: MUST be either 'STX' or a valid SIP-010 contract address.
+
+- amount: The amount to be paid.
+
+**Optional Parameters:**
+
+- description: MAY be included to provide context for the payment request.
+
+- expiresAt: MAY be included; if included, wallets MUST NOT process payment requests past the expiry time.
+
+- invoiceNumber: MAY be included as an identifier.
+
+- dueDate: MAY be included.
+
+- memo: MAY be included. MUST NOT include personal information.
+
+**Parameters to Ignore:**
+
+contractName, functionName: Wallets MUST ignore these parameters if present.
+
+### `mint`
+
+**Function:** Represents a non-fungible token minting request.
+
+**Required Parameters:**
+
+- operation: 'mint'
+
+- contractName: A valid SIP-009 contract address.
+
+- functionName: A public contract function, typically `mint` or `claim`
+
+- token: MUST be either 'STX' or a valid SIP-010 contract name.
+
+- amount: The amount to be paid.
+
+**Optional Parameters:**
+
+- recipient: MAY be included; if present, MUST be a valid Stacks address. The NFT will be minted for the specified address.
```

Comment:
Where is it minted if this is not specified?

Reply:
By default, the mint recipient is the user paying for the mint.  If you include a recipient, you mint for that recipient.